Solutions/InstanceManifests/Development.json

```diff
@@ -1,13 +1,16 @@
 {
   "services": {
     "Marain.Tenancy": {
-      "release": "0.3.0-59-deployment-advertise-service.9"
+      "omit": false,
+      "release": "0.3.0-powershell-deploy.37"
```

Comment:
We should probably get the PRs landed in these other projects first, then update this to reference at least a preview build from master rather than a feature branch build.